testing/test_mixup.py

```diff
@@ -0,0 +1,26 @@
+import ivadomed.utils as imed_utils
+import torch
+
+
+def test_mixup():
+    inp = [[[[0 for i in range(40)] for i in range(40)]]]
+    targ = [[[[0 for i in range(40)] for i in range(40)]]]
+    for i in range(10):
+        for j in range(10):
+            targ[0][0][i][j]=1
+    inp = torch.tensor(inp).float()
+    targ = torch.tensor(targ).float()
+    # just testing if mixup function run
+    out = imed_utils.mixup(inp, targ, alpha=0.5)
+
+
+def test_mixup_save():
```

Comment:
@lrouhier --> What about combining these 2 two in just one test function with a `@pytest.mark.parametrize`?

Reply:
Right good idea!